templates/home/index.html

```diff
@@ -12,7 +12,15 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.4.0/css/font-awesome.min.css">
     <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/@glidejs/glide/dist/css/glide.theme.min.css">
 {% endblock %}
+{% style box for email%}
```

Comment:
This tag doesn't work in Django. You need to make the required changes in `static/home/css/style.css` file. Also, keep in mind to put the style in the correct section (sections are arranged in the order they appear on the home page.